platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: "Data studios deployment"
+description: Deploy Seqera Platform with Data studios
+date: "12 Jun 2024"
+tags: [docker, compose, kubernetes, data, studios, deployment]
+---
+
+This guide assumes that all prerequisites have been met. Visit the corresponding **Prerequisites** page for your infrastructure provider.
+
+## Docker compose
+
+To run data studios in docker compose setup uncomment following values in the `_templates/docker/data-studios.env` file:
+
+```dotenv
+# Data studios settings
+TOWER_DATA_STUDIO_ALLOWED_WORKSPACES=<ALLOWED WORKSPACES>
+
+# Enable Data Explorer to enable mounting data to your data studios
+TOWER_DATA_EXPLORER_ENABLED=true
+TOWER_DATA_EXPLORER_CLOUD_DISABLED_WORKSPACES=
+
+TOWER_DATA_STUDIO_CONNECT_URL=<YOUR CONNECT URL, SAME AS `CONNECT_PROXY_URL` IN `data-studios.env` >
+TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN=<YOUR INITIAL ACCESS TOKEN, IDEANTICAL AS `INITIAL_ACCESS_TOKEN` in `data-studios.env`>
+TOWER_OIDC_PEM_PATH=<PATH TO PEM FILE>
+```
+
+fill `_templates/docker/data-studios.env` values and uncomment following lines in the `_templates/docker/docker-compose.yml` file:
+
+```yaml
+  connect-proxy:
+    image: cr.seqera.io/private/nf-tower-enterprise/data-studio/tower-connect-proxy:0.6.14
+    platform: linux/amd64
+    env_file:
+      - data-studios.env
+    networks:
+      - frontend
+      - backend
+    ports:
+      - 9090:9090
+    restart: always
+    depends_on:
+      - redis
+
+  connect-server:
+    image: cr.seqera.io/private/nf-tower-enterprise/data-studio/tower-connect-server:v0.6.14
+    platform: linux/amd64
+    env_file:
+      - data-studios.env
+    networks:
+      - backend
+    ports:
+      - 7070:7070
+    restart: always
+```
+
+Data studios RSA key is required for the data studios functionality. Generate your key and provide the path to it in the
+`TOWER_OIDC_PEM_PATH` variable in the `data-studios.env` file. Next mount the pem file in the `backend` service in the `docker-compose.yml` file.
+
+See example in the configuration with `data-studios-rsa.pem` file mounted in the `backend` service:
+```yaml
+  backend:
+    volumes:
+      - $PWD/data-studios-rsa.pem:/data-studios-rsa.pem
+```
+
+## Kubernetes
+TBD
+
+## DNS setup
+Proxy domain (`CONNECT_PROXY_URL`) has to be a subdomain of main platform domain (example: if platform domain is
+`tower.io`, then proxy domain should be `connect.tower.io`).
+Data studios require a wildcard DNS record that will match request for non-existent subdomains.
+It is specified by a `*` as part of the domain name, example: `*.tower.io`. It should be pointed to the port 9090.
```

Comment:
They also need a valid wildcard ssl certificate to cover the third level domain.

We might want to suggest to use only a third level domain if possible, so that they can use a cheaper certificate.

Reply:
Great catches! Thanks for the comments, I will change that now.